### ISSUE TITLE
1.Calcularea matricii de confuzie la pragul ales. Ștefania(Gr.B)

### ISSUE THREAD

**Issue by licastefania200001-alt** [georgerusu017/churn_rate_predicition#43]

**Comment by licastefania200001-alt** [comment]
import pandas as pd
from sklearn.metrics import confusion_matrix, accuracy_score, precision_score

df = pd.read_csv("/Users/stefanialica/Documents/PYTHON/Task-uri/tranzactii.csv", sep='\t')
# ELIMINAREA SPATIILOR
df.columns = df.columns.str.strip()  
print("Coloane disponibile:", df.columns.tolist())

df = df.dropna(subset=['Class', 'f1','f2','f3','f4','f5'])
y_true = df['Class'].tolist()  # 1 = frauda, 0 = normal
y_score = df[['f1','f2','f3','f4','f5']].sum(axis=1).tolist()
# PRAGUL
threshold = 0.5
y_pred_threshold = [1 if s >= threshold else 0 for s in y_score]
# CALCULAM MATRICEA DE CONFUZIE
cm = confusion_matrix(y_true, y_pred_threshold)
print("Matrice de confuzie la pragul", threshold,)
print(cm)
accuracy = accuracy_score(y_true, y_pred_threshold)
precision = precision_score(y_true, y_pred_threshold)
print(f"Acuratete: {accuracy:.4f}")
print(f"Precizie : {precision:.4f}")
